[2021-09-29 17:50:58,820] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: test-05.set_marriage_status 2021-09-29T20:20:00+00:00 [queued]>
[2021-09-29 17:50:58,835] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: test-05.set_marriage_status 2021-09-29T20:20:00+00:00 [queued]>
[2021-09-29 17:50:58,835] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2021-09-29 17:50:58,835] {taskinstance.py:881} INFO - Starting attempt 1 of 2
[2021-09-29 17:50:58,835] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2021-09-29 17:50:58,860] {taskinstance.py:901} INFO - Executing <Task(PythonOperator): set_marriage_status> on 2021-09-29T20:20:00+00:00
[2021-09-29 17:50:58,863] {standard_task_runner.py:54} INFO - Started process 3896 to run task
[2021-09-29 17:50:58,915] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'test-05', 'set_marriage_status', '2021-09-29T20:20:00+00:00', '--job_id', '238', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/test05.py', '--cfg_path', '/tmp/tmp9u2me_7_']
[2021-09-29 17:50:58,917] {standard_task_runner.py:78} INFO - Job 238: Subtask set_marriage_status
[2021-09-29 17:50:59,041] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: test-05.set_marriage_status 2021-09-29T20:20:00+00:00 [running]> alunoigti-VirtualBox
[2021-09-29 17:50:59,513] {python_operator.py:114} INFO - Done. Returned value was: None
[2021-09-29 17:50:59,515] {taskinstance.py:1057} INFO - Marking task as SUCCESS.dag_id=test-05, task_id=set_marriage_status, execution_date=20210929T202000, start_date=20210929T205058, end_date=20210929T205059
[2021-09-29 17:51:03,820] {local_task_job.py:102} INFO - Task exited with return code 0
